coordinator/src/services/translation/request/send-message-payload.ts

```diff
@@ -132,24 +133,19 @@ function createControlActEventAuthor1(asid: string) {
   return new hl7V3.SendMessagePayloadAuthorSystemSds(agentSystemSds)
 }
 
-function createReleaseControlActEvent<T>(
+function createControlActEventForUnattendedAccess<T>(
   subject: T
 ) {
   const controlActEvent = new hl7V3.ControlActEvent<T>()
-  controlActEvent.author = convertRequesterToReleaseControlActAuthor(subject)
+  controlActEvent.author = createControlActAuthorForUnattendedAccess()
   controlActEvent.author1 = createControlActEventAuthor1(process.env.FROM_ASID)
   controlActEvent.subject = subject
   return controlActEvent
 }
 
-function convertRequesterToReleaseControlActAuthor<T>(
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  hl7ReleaseRequest: T
-) {
+function createControlActAuthorForUnattendedAccess() {
   const sdsUniqueIdentifier = "G9999999"
-
-  const sdsJobRoleCode = "R8000"
-
-  const sdsRoleProfileIdentifier = "100102238986"
+  const sdsJobRoleCode = "R9999"
+  const sdsRoleProfileIdentifier = "999999999999"
```

Comment:
I've created a pr for dispense-notification which uses this and moved these hardcoded values into the constuctor for classes prefixed with `Unattended` i.e. `UnattendedSdsJobRoleCode` etc so they can be reused, we can find references and the values are consistent across the codebase

Reply:
Will chat to you about how to resolve before merging

Reply:
Will tidy up unattended stuff in a new PR